### PR TITLE
First Capstone project

### PR DESCRIPTION
**Pull request by PeteUgwu**
## Loom video link https://www.loom.com/share/14102be80718405487fd263035ed3751

- [ ] Applied css, html and javascript best practices
- [ ] Personalized the design contents 
- [ ] followed the design guidelines
- [ ] Built home and about pages
- [ ] Created featured speakers session dynamically
- [ ] fixed linters error
- [ ] Implemented basic navigations and pages interactions
- [ ] Updated READme file

### CODE REVIEW

**Comment by Nyame-Wolf** [comment]
> ## Loom video link https://www.loom.com/share/14102be80718405487fd263035ed3751


Good explanation about your project, and the most exciting thing about it however, kindly note that the maximum length of the video should be 5 mins. Your's exceeds by 28 seconds. Please trim it and remove some parts to meet the requirements
Screenshot 2022-10-29 at 17 01 28

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
Comment (Nyame-Wolf) [inline .vscode/settings.json:3]: [OPTIONAL] I suggest you add this to the .gitignore file This represents the local settings for your vscode.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        </nav>
+    
+      <main id="main">
+    <div class="landing">
+      <p class="before-h12">"Hello! Food Lovers"</p>
+      <h1>GTCO ANNUAL FOOD AND DRINK FESTIVAL 2022 </h1>
+      <p class="description-container">
+        GTCO Food & Drink Fair is a an annual food exhibition and sales event aimed at projecting the diverse angles of the food industry.
+      </p>
+      <h2>2022.04.30 (Wed) ~ 2ND MAY (Sat) <br><p class="paragraph">@ GT Centre Plot 1, Water Corporation Drive, Oniru, VI, Lagos.</p></h2>
```
Comment (Nyame-Wolf) [inline index.html:66]: I suggest you activate the transparent background for the mobile version too for consistency in your project. Currently your the mobile version has no transparent background
This is your desktop version with the transparent background:
Screenshot 2022-10-29 at 17 11 25
This is your mobile version 
Screenshot 2022-10-29 at 17 11 20

Please note as per the design, they both have a transparent background:
Screenshot 2022-10-29 at 17 11 01

File: index.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        <ul class="flex nav-list-item">
+            <li class="nav-item nav-about"><a href="about.html">About</a></li>
+            <li class="nav-item nav-color"><a href="#">Exhibition</a></li>
+            <li class="nav-item nav-color"><a href="#">Join</a></li>
+            <li class="nav-item nav-color"><a href="#">Sponsor</a></li>
+            <li class="nav-item nav-color"><a href="#">Schedules</a></li>
+            <li class="nav-item nav-border"><a href="#program">THE CHEFS</a></li>
+        </ul>
+    </div>
+    </header>
```
Comment (Nyame-Wolf) [inline index.html:36]: I suggest you add the top navigation as in the design by cindy with the social media links in your desktop version. Kindly note that the header has two nav bars:
Screenshot 2022-10-29 at 17 11 01

File: script.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+const loadSpeakers = (item) => {
+  let display = ' ';
+  const data = isDesktop || isShowMore ? item : !isDesktop && item.slice(0, 2);
+  data.forEach(({
+    name, featuredImage, post, details,
+  }) => {
+    display += `
+      
+    
+        <div class="speaker-info">
```
Comment (Nyame-Wolf) [inline script.js:83]: Kindly use the correct semantic tag for this. For this project, the requirement is that you match `<section> with <article>`
A div is a general element. Having these as articles are good for semantics and also screen readers.

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @PeteUgwu  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the same  design ☑️ 
- Linters are passing .
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by danielufeli — APPROVED** [review]
Hi @PeteUgwu ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**